modules/admin_manual/pages/release_notes.adoc

```diff
@@ -95,18 +95,21 @@ For more information on the Media Viewer app, visit the https://doc.owncloud.com
 === OAuth2 and session handling improvements
 
 Server 10.3 comes with improvements for session handling with Redis.
-These are designed to cope with issues encountered around duplicate session tokens, which make the ownCloud Clients lose their OAuth2 authorization from time to time, and forcing users to re-authenticate. 
-The session handling stability has been generally improved, and support for Redis Session Locking has been introduced. 
+These are designed to cope with issues encountered around duplicate session tokens, which make the ownCloud Clients lose their OAuth2 authorization from time to time, and forcing users to re-authenticate.
```

Comment:
The final clause in this sentence doesn't read correctly to me. I suggest changing it to ", which forces users to re-authenticate".